runtime/pangolin/src/pallets/deposit.rs

```diff
@@ -34,7 +39,7 @@ impl darwinia_deposit::SimpleAsset for PKtonMinting {
 			Err(<pallet_assets::Error<Runtime>>::BalanceLow)?;
 		}
 
-		Assets::burn(RuntimeOrigin::signed(ROOT), asset_id, *who, amount)
+		Assets::burn(RuntimeOrigin::signed(ROOT), codec::Compact(asset_id), *who, amount)
```

Comment:
`codec::Compact(asset_id)` => `asset_id.into()`. There are many places you need to update.